### PR TITLE
Events log (user added a track)

### PR DESCRIPTION
**Pull request by adrianmoya**
This PR adds a new WS endpoint for events and bounces the addTrackEvent to clients so the FE can show a log of songs added by other people connected.

The events socket may be used in the future for other purposes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+
+## Requirements.
+
+1. You must have a local instance of mongoDB runing in your local environment or:
+
+`docker run --name mongo -d -p27017:27017 mongo:3.4`
+
+## Run in dev with:
+
+`java -jar ./target/soundfoundry-0.0.1-SNAPSHOT.jar --spring.profiles.active=dev`
```
Comment (Efgar) [inline README.md:18]: +1!!!

File: src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	
+	@MessageMapping("/events")
+	@SendTo("/topic/events")
+	public Event processEvent(Event event){
+		log.info("Entering this shit");
+		log.debug("Event: {}", event);
+		switch(event.getType()){
+		
+		case ADDTRACK:
+			//Track track = new Track();
```
Comment (Efgar) [inline src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java]: Remove comments?

File: src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+	@Autowired
+	private RoomService roomService;
+	
+	private Logger log = LoggerFactory.getLogger(EventsController.class);
+	
+	@MessageMapping("/events")
+	@SendTo("/topic/events")
+	public Event processEvent(Event event){
+		log.info("Entering this shit");
```
Comment (Efgar) [inline src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java]: Remove this logger!!

File: src/main/java/com/tfflabs/soundfoundry/dto/Event.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
+package com.tfflabs.soundfoundry.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gson.Gson;
+
+public class Event {
```
Comment (Efgar) [inline src/main/java/com/tfflabs/soundfoundry/dto/Event.java]: @ Data elimina la mitad del boilerplate si no todo en esta clase

**Review by Efgar — CHANGES_REQUESTED** [review]

File: src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java (hunk 3 above)
Comment (adrianmoya) [reply src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java]: LOL

File: src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java (hunk 2 above)
Comment (adrianmoya) [reply src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java]: Yeah, that was only to talk with you guys about the possibilities....

**Review by Efgar — APPROVED** [review]

**Comment by Efgar** [comment]
LGTM

**Pull request merged**